packages/db/src/test/utils.ts

```diff
@@ -21,10 +22,10 @@ const tempDir = tmp.dirSync({ unsafeCleanup: true });
 tmp.setGracefulCleanup();
 
 export class WorkspaceClient {
-  private workspace: Workspace;
+  private workspace: Databases;
```

Comment:
In keeping with removing workspace, should we rename this? Fine either way of course since it doesn't affect the root goal of removing the subschema.

Reply:
Ah perhaps. I'm not totally happy with `Databases` as a name, so I liked the idea of keeping `workspace` as the context variable. Do you prefer calling it `databases`?

Reply:
Hm. I mean what is it really that we're working with here? I suppose `databases` is accurate. `rootDatabases`? I'm terrible at coming up with names. It isn't a big deal, I guess I just thought someone new to the code would be like "what is a workspace in this context?"

Reply:
Hm, I'm torn. Your argument is compelling and you may have won me over, but let me ask you what you prefer, for writing resolvers:

```typescript
resolve: ({ id }, _, { databases }) => 
  databases.get("sources", id)
```

```typescript
resolve: ({ id }, _, { workspace }) => 
  workspace.get("sources", id)
```

Reply:
How bout this alternative: we rename `interface Databases` to `interface Workspace`? We can keep all the classes having the same name; we'd just do `class Databases<C extends Collections> implements Workspace<C extends Collections>`

Reply:
Alright I went the other direction like I proposed. Let me know what you think!

Reply:
Oops sorry I missed your comments! `interface Workspace` seems fine!

Reply:
And this way there's no need to change the resolvers, right?

Reply:
Exactly!